test/UID2.Client.Test.Utils/UID2TokenGenerator.cs

```diff
@@ -87,7 +106,16 @@ public static string GenerateEUIDTokenV3(string uid, Key masterKey, int siteId,
             {
                 IdentityScope = (int) IdentityScope.EUID
             };
-            return GenerateUID2TokenV3(uid, masterKey, siteId, siteKey, param);
+            return GenerateUID2TokenWithDebugInfo(uid, masterKey, siteId, siteKey, param, false);
+        }
+        
+        public static string GenerateEUIDTokenV4(string uid, Key masterKey, int siteId, Key siteKey)
```

Comment:
can remove this version, is there GenerateEUIDTokenV4 that takes Params?

Reply:
fixed